tests/smoke_test.go

```diff
@@ -0,0 +1,136 @@
+package tests
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	ptesting "github.com/pulumi/pulumi/sdk/v3/go/common/testing"
+	"github.com/stretchr/testify/require"
+)
+
+var Runtimes = []string{"python", "java", "go", "yaml", "nodejs", "dotnet"}
+
+// Mapping from the language runtime names to the common language name used by templates and the like.
+var Languages = map[string]string{
+	"python": "python",
+	"java":   "java",
+	"go":     "go",
+	"yaml":   "yaml",
+	"nodejs": "typescript",
+	"dotnet": "csharp",
+}
+
+// Quick sanity tests for each downstream language to check that a minimal example can be created and run.
+//
+//nolint:paralleltest // pulumi new is not parallel safe
+func TestLanguageNewSmoke(t *testing.T) {
+	for _, test := range Runtimes {
+		tt := test
```

Comment:
minor
The loop variable and the reassignment should be the same (typically `tt`).
The idea is that the copy should shadow the loop variable
so that we never accidentally refer to it inside the parallel code.

Although in this case, the loop is not parallel so we probably don't need this at all.

Reply:
Ah I remember thinking something about this looked odd when I wrote it but couldn't put my finger on why, but was probably this. I'll fix it up and have a quick look if we've got any other inconsistencies I might have copied this from.